src/hooks/useSidePanelDisplayStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import useLocalize from './useLocalize';
+import useOnyx from './useOnyx';
+import useResponsiveLayout from './useResponsiveLayout';
+
+/**
+ * Hook to get the display status of the Side Panel
+ */
+function useSidePanelDisplayStatus() {
```

Comment:
I think you could add it to the context too, so that the logic is not running multiple times. Wdyt?